packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.js

```diff
@@ -59,11 +54,21 @@ const PriceSummary = props => {
 
     const { subtotal, total, discounts, giftCards, taxes, shipping } = flatData;
 
-    const priceClass = isCartUpdating ? classes.priceUpdating : classes.price;
-    const totalPriceClass = isCartUpdating
+    const priceClass = isUpdating ? classes.priceUpdating : classes.price;
```

Comment:
I understand this is not on you, but can you change the style of css class names to `underscore casing` from `camel casing`? I am not sure if we have a standard on that, but we should make a choice here for consistency.

Reply:
There are many places where this is happening, and we have makes changes incrementally once we decide on what is the way to go.

Reply:
I tend to write classes with camel case. Most of the underscore names you see are from the original scaffolding of the checkout page where we didn't call out the naming convention of css classes.

We can discuss as a team and add something to the wiki.

Reply:
Correct. Classnames should, preferably, be camelcased.